context.go

```diff
@@ -48,25 +49,57 @@ func NewContext(rootLevel Level) *Context {
 // its parents if necessary.
 func (c *Context) GetLogger(name string) Logger {
 	name = strings.TrimSpace(strings.ToLower(name))
+
 	c.modulesMutex.Lock()
 	defer c.modulesMutex.Unlock()
-	return Logger{c.getLoggerModule(name)}
+
+	return Logger{
+		impl: c.getLoggerModule(name),
+	}
 }
 
-func (c *Context) getLoggerModule(name string) *module {
+func (c *Context) getLoggerModule(name string) module {
 	if name == rootString {
 		name = ""
 	}
+	// Located the exact module by name
 	impl, found := c.modules[name]
 	if found {
 		return impl
 	}
-	parentName := ""
+	// Expand any wildcards to find the module.
+	if strings.Contains(name, "*") {
+		if re, err := regexp.Compile(strings.ReplaceAll(name, "*", "\\w+[^.]")); err == nil {
+			var modules []module
+			for name, module := range c.modules {
+				if re.MatchString(name) {
+					modules = append(modules, module)
+				}
+			}
+			if len(modules) == 0 {
+				// Not sure what to do here?
+				panic("whoops")
```

Comment:
What's the best way to solve this, as we really don't want to panic.

Return the parent logger of the wildcard?